libexec/bats-core/bats

```diff
@@ -80,6 +84,98 @@ expand_path() {
   printf -v "$result" '%s/%s' "$dirname" "${path##*/}"
 }
 
+# argument: exclude_literal
+is_bats_file() {
+  [[ "$1" == *".bats"* ]]
+}
+
+# argument suite_filename
+is_excluded() {
+  if [[ -z "$exclude_files" ]]; then
+    return 1
+  fi
+  
+  local suite_filename="$1"
+  read -ra EXCLUDE_FILES <<< "$exclude_files"
```

Comment:
I'd prefer explicitly passed parameters where possible. Can we use $2 onwards instead of $exclude_files?

Reply:
yes, we can :-) a matter of taste.